src/main/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManager.java

```diff
@@ -621,4 +622,17 @@ public void run() {
       syncStorage();
     }
   }
+
+  public Map<String, String> getRemoteProjects(String connectionId) {
+    var serverConfiguration = getServerConfigurationFor(connectionId);
```

Comment:
Should this maybe be surrounded with a `try..catch` ? And also return an empty map in the catch block. Because I've seen a TimeOut exception thrown from the getConfiguration method, I think `could not get settings in time` or similar...

Reply:
I believe this is only supposed to happen if there was an error during the LS startup. I suggest we merge this and see if it happens frequently in dogfood.